Contribute/Code-samples/index.md

```diff
@@ -0,0 +1,131 @@
+---
+meta.Title: "Code samples - Best practices"
+meta.Description: "We highly recommend supporting your documentation articles with code samples. Make sure the samples follow our best practices outlined in this article."
+---
+
+# Code samples
+
+The articles in the Umbraco Documentation can in most cases benefit from relevant code samples to support the written text.
+
+In this article you will find some best practices which outline how we recommend formatting and using code samples, and you will also find definitions and examples of the most used types of code samples in the Umbraco Documentation.
+
+## Best practices
```

Comment:
I'd avoid the term 'Best Practices' wherever possible - Guidelines, and 'Good Practices' are better alternatives, 'best' makes the statement that the thing cannot be improved, it is best, cannot be bettered, but we're totally open to improvement!

Reply:
That's a good point 🤔
I'll update to "Good practices" instead!